contrib/strongswan/template.py

```diff
@@ -0,0 +1,112 @@
+pkgname = "strongswan"
+pkgver = "5.9.14"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--libexecdir=/usr/lib",
+    "--with-ipsecdir=/usr/lib/strongswan",
+    "--with-capabilities=libcap",
+    "--with-user=_ipsec",
```

Comment:
does it make sense to have a "generic" user name for this instead of something like `_strongswan`?

(i don't know, please explain your thoughts here)

Reply:
No reason at all.  I'm going to change it to `_strongswan`